hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/hdds/scm/container/balancer/TestContainerBalancer.java

```diff
@@ -127,31 +128,26 @@ public void testStartBalancerStop() throws Exception {
     startBalancer(balancerConfiguration);
     try {
       containerBalancer.startBalancer(balancerConfiguration);
-      Assertions.assertTrue(false,
-          "Exception should be thrown when startBalancer again");
+      Assertions.fail("Exception should be thrown when startBalancer again");
```

Comment:
I think we should simplify this and use assertThrow. 
I looked at sub tasks of HDDS-9952 but didn't see any sub tasks. May be create one if you don't have already.

Reply:
Thanks for the suggestion.  I also planned to create another task for `assertThrows`, now filed HDDS-9956.

Creating these as top-level issues so that we can create sub-tasks for specific modules if needed (to avoid too large patches).